### DISCUSSION TITLE
ignore darkreader in some css selector ?

### DISCUSSION THREAD

**Discussion by Tonysmark** [darkreader/darkreader#10827] [Q&A]
darkreader doing some greatest work in my web site, but some dom element in html, I dont wan't to reverse it, and i'm using `darkreader.js` , I was wonder maybe there is some custom selector in dark reader that can disable or ignore by the analysis engine, such as `.darkreader--ignore` ?

**Comment by bershanskiy** [comment]
Hi, Dark Reader npm library (which I think you are referring to) supports custom configurations. Please refer to our TS declarations, specifically the second (optional) arguments called `fixes` for `enable()` and `auto()`. If you have any questions, please ping me.

**Reply by Tonysmark** [reply]
Sure, I am using npm library (sorry), I was using <DynamicThemeFix> , and write I css selector such as
```js
endable({...},{css:'.button { color: #e62828 } '})
```

but it seem like no thing happed 

what's wrong with it ? Please tell me how to do it

**Reply by bershanskiy** [reply]
This is a CSS selector specificity issue, so I'd recommend one of two hings:
1. use [`!important` CSS declaration](https://developer.mozilla.org/en-US/docs/Web/CSS/important) to override specificity in the cascade
2. use a higher specificity selector, e.g., one with `id` attribute (`id=something` or `#something`)

I can't provide a more specific recommendation unfortunately without seeing the page (if it's public).
